packages/react/src/Radio/RadioGroup.tsx

```diff
@@ -58,27 +75,55 @@ export const RadioGroup = forwardRef<HTMLDivElement, RadioGroupProps>(
       [defaultValue, isControlled, name, onChange, required, value],
     );
 
+    const ownRef = useRef(null);
+
+    const { validity, validationMessage } = useFormControlValidity(
+      ownRef,
+      !disableValidation,
+    );
+
+    const id = useFallbackId(idProp);
```

Comment:
Når du genererer den id-en her burd ekanskje `groupId` slettes og erstattes av denne. Ingen hensikt å generere to ider.